e2e-tests/insurance/cypress/e2e/journeys/your-business/check-your-answers/check-your-answers-summary-list/check-your-answers-summary-list.spec.js

```diff
@@ -37,68 +40,106 @@ context('Insurance - Your business - Check your answers - Summary list - your bu
   let referenceNumber;
   let url;
 
-  before(() => {
-    cy.completeSignInAndGoToApplication({}).then(({ referenceNumber: refNumber }) => {
-      referenceNumber = refNumber;
+  describe(`${TRADING_ADDRESS} as no`, () => {
+    before(() => {
+      cy.completeSignInAndGoToApplication({}).then(({ referenceNumber: refNumber }) => {
+        referenceNumber = refNumber;
 
-      cy.startYourBusinessSection();
+        cy.startYourBusinessSection();
 
-      cy.completeAndSubmitCompanyDetails({});
-      cy.completeAndSubmitNatureOfYourBusiness();
-      cy.completeAndSubmitTurnoverForm();
-      cy.completeAndSubmitCreditControlForm({});
+        cy.completeAndSubmitCompanyDetails({});
+        cy.completeAndSubmitNatureOfYourBusiness();
+        cy.completeAndSubmitTurnoverForm();
+        cy.completeAndSubmitCreditControlForm({});
 
-      url = `${baseUrl}${ROOT}/${referenceNumber}${CHECK_YOUR_ANSWERS}`;
+        url = `${baseUrl}${ROOT}/${referenceNumber}${CHECK_YOUR_ANSWERS}`;
+      });
     });
-  });
 
-  beforeEach(() => {
-    cy.saveSession();
+    beforeEach(() => {
+      cy.saveSession();
 
-    cy.navigateToUrl(url);
-  });
+      cy.navigateToUrl(url);
+    });
 
-  after(() => {
-    cy.deleteApplication(referenceNumber);
-  });
+    after(() => {
+      cy.deleteApplication(referenceNumber);
+    });
 
-  it(`should render a ${HAS_DIFFERENT_TRADING_NAME} summary list row`, () => {
-    checkSummaryList[HAS_DIFFERENT_TRADING_NAME]();
-  });
+    it(`should render a ${HAS_DIFFERENT_TRADING_NAME} summary list row`, () => {
+      checkSummaryList[HAS_DIFFERENT_TRADING_NAME]();
+    });
 
-  it(`should render a ${TRADING_ADDRESS} summary list row`, () => {
-    checkSummaryList[TRADING_ADDRESS]();
-  });
+    it(`should render a ${TRADING_ADDRESS} summary list row`, () => {
+      checkSummaryList[TRADING_ADDRESS]();
+    });
 
-  it(`should render a ${WEBSITE} summary list row`, () => {
-    checkSummaryList[WEBSITE]();
-  });
+    it(`should not render a ${FULL_ADDRESS} summary list row`, () => {
+      checkSummaryList[FULL_ADDRESS](false);
```

Comment:
What's the difference between this "false" flag and the new `assertSummaryListRowDoesNotExist` command?

Reply:
wanted to keep this test file constant.  So when you call checkSummaryList, it is able to handle this conditional row